src/store/modules/search.js

```diff
@@ -0,0 +1,21 @@
+import axios from 'axios'
+import { API_KEY, path } from '../../api/tmdb-api'
+
+
+export const search = {
```

Comment:
Please, set the `namespaced` property to this module. Details you can check in Vuex doc: https://vuex.vuejs.org/guide/modules.html#namespacing